### DISCUSSION TITLE
[Question] Convert iconomy to cmi

### DISCUSSION THREAD

**Discussion by Malachiel87** [Zrips/CMI#953] [Enhancement]
Hi i want know if is possible to import iconomy data to cmi, i use offline mode, is possible?

**Comment by CloudeLecaw** [comment]
No it is not we also had this Problem. and the answer is easy
Change to CMI and manually Change the 100000 Players to your new mysql table or edit them in the SQLite database.

a convert command does not exists.
Ma you can ask a developer to develop a tool for this.

**Comment by Malachiel87** [comment]
what if i use essentials for some weeks to take player data to hook to vaulteconomy and then use cmi import data?

**Comment by CloudeLecaw** [comment]
u asked for iconomy data Import not for Import of essentials data….
iconmy is an other pugin..
we also had this Problem when we changed from iconomy to CMIs economy.

**Comment by Malachiel87** [comment]
essentials can hook in any economy, importing cmi from essentials it will import also iconomy data? or i am wrong?

**Comment by CloudeLecaw** [comment]
yes to beh oney this is wrong.
economy is ist own Plugin.

you have two Options:
1. use your economy with CMI
2. manually Transfer all economy data from your iconomy to CMIs one.

both works.
the difference is just if you use the first one then you will have an extra remaining Plugin for your economy.


Essentials self can hook into this with the CMI enjector or with the Special Vault Version on the CMi page.

**Comment by Malachiel87** [comment]
so i think i will do this, use cmi for some months and then move all data via custom migrator

**Comment by CloudeLecaw** [comment]
good try, would be nice to hear the results :)

**Comment by Malachiel87** [comment]
Yeah I done that you can close it
